CIP-0013/CIP-0013.md

```diff
@@ -75,10 +79,15 @@ paymentref = cardanoaddress [ "?" amountparam ]
 cardanoaddress = *(base58 | bech32)
 amountparam = "amount=" *digit [ "." *digit ]
 
-stakepoolref = "//stake?" stakepool
+stakepoolref = "//stake" stakequery
+stakequery = ( "?" stakepoolpair) *( "&" stakepoolpair)
+stakepoolpair = stakepool [ "=" proportion]
+
 stakepool = poolhexid | poolticker
 poolhexid = 56HEXDIG
 poolticker = 3*5UNICODE
+
+proportion = *digit [ "." *digit ]
```

Comment:
If anything, it'd be good to have CIP-0017 & this extension aligned; Please see [the comment on CIP-0017 about decimals](https://github.com/cardano-foundation/CIPs/pull/82#issuecomment-827468701)

Reply:
thanks, addressed here: https://github.com/cardano-foundation/CIPs/pull/82#issuecomment-830012606